uqcsbot/scripts/latex.py

```diff
@@ -0,0 +1,42 @@
+from uqcsbot import bot, Command
+from urllib.parse import quote
+import re
+
+
+def handle_latex_internal(channel, data):
+    url = f"http://latex.codecogs.com/gif.latex?{quote(data)}"
+    bot.post_message(
+        channel, 
+        text=f"LaTeX render for \"{data}\"",
+        attachments=[
+            {
+                "fallback": f"Codecogs LaTeX image for {data}",
+                "image_url": url,
+                "from_url": url,
+                "id": 1,
```

Comment:
what's `id` used for? (is it required?)

Reply:
No idea, I think it's just for ordering? I copied it from a deconstructed `message_changed` event.

Reply:
strip out as much as possible imo, if it ain't needed turf it